scripts/update_zenodo.py

```diff
@@ -0,0 +1,180 @@
+                if calculate_md5(file_path)!=checksum:
+                    logger.info(filename, calculate_md5(file_path),remote_files[filename])
+                    
+                    # replace/update existing file
+                    response = requests.delete(f'https://zenodo.org/api/deposit/depositions/{new_deposition_id}/files/{file_id}',
+                                                params={'access_token': ACCESS_TOKEN})
+                    if response.status_code == 204:
+                        logger.info(f"Old File deleted. {filename}")
+                    else:
+                        logger.info(f"Failed to Remove old {filename}. Status code: {response.status_code}")
+                        logger.info(json_response)
+                        rm_draft(ACCESS_TOKEN, new_deposition_id)
+
+                    url = f'https://zenodo.org/api/deposit/depositions/{new_deposition_id}/files?access_token={ACCESS_TOKEN}'
+                    data = {'name': filename}
+                    files = {'file': open(file_path, 'rb')}
+                    response = requests.post(url, data=data, files=files)
+                    if response.status_code == 201:
+                        logger.info(f"File added to the new draft. {filename}")
+                        to_be_updated += [filename]
+                    else:
+                        logger.info(f"Failed to add {filename} to the new draft. Status code: {response.status_code}")
+                        logger.info(response.json())
+                        rm_draft(ACCESS_TOKEN, new_deposition_id)
+            else:
+                logger.info(f"new file :{filename}")
+                # upload new file
+                url = f'https://zenodo.org/api/deposit/depositions/{new_deposition_id}/files?access_token={ACCESS_TOKEN}'
+                data = {'name': filename}
+                files = {'file': open(file_path, 'rb')}
+                response = requests.post(url, data=data, files=files)
+                if response.status_code == 201:
+                    logger.info(f"File added to the new draft. {filename}")
+                    to_be_updated += [filename]
+                else:
+                    logger.info(f"Failed to add {filename} to the new draft. Status code: {response.status_code}")
+                    logger.info(response.json())
+                    rm_draft(ACCESS_TOKEN, new_deposition_id)
+
+        if len(to_be_updated)<1:
+            logger.info("No updates detected")
+            rm_draft(ACCESS_TOKEN, new_deposition_id)
+        else:
+            # publish draft
+            url_publish = f"https://zenodo.org/api/deposit/depositions/{new_deposition_id}/actions/publish"
+            response = requests.post(url_publish, params={'access_token': ACCESS_TOKEN})
+
+            if response.status_code == 202:
+                logger.info(f"New version published. ID: {new_deposition_id}")
+            else:
```

Comment:
I think we need to create the `success.log` file here?

Reply:
somehow missed that, added `success.log` at line 149